src/pybind/mgr/dashboard/HACKING.rst

```diff
@@ -185,7 +185,7 @@ Note::
    `protractor.conf.js` to point to the appropriate destination.
 
 Making code reuseable
-"""""""""""""""""""""
+~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Was this it? :scream:

Reply:
Just curious, how did you find this? I run the whole HACKING.rst through a reStructuredText validator and it didnt' report anything.

Reply:
Yes, that's the reason.

To be honest: it was just a feeling/intuition...